docker/builder/tools.Dockerfile

```diff
@@ -28,10 +28,7 @@ RUN cd /usr/local/bin && wget "https://storage.googleapis.com/kubernetes-release
 COPY --link --from=tools-builder /aptos/dist/aptos-db-bootstrapper /usr/local/bin/aptos-db-bootstrapper
 COPY --link --from=tools-builder /aptos/dist/aptos-db-tool /usr/local/bin/aptos-db-tool
 COPY --link --from=tools-builder /aptos/dist/aptos /usr/local/bin/aptos
-COPY --link --from=tools-builder /aptos/dist/aptos-openapi-spec-generator /usr/local/bin/aptos-openapi-spec-generator
```

Comment:
Unfortunately removing this will cause CI to fail, we use this in this workflow: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/node-api-compatibility-tests.yaml#L92.